aha-table.html

```diff
@@ -55,23 +55,22 @@
         <!-- Second Header (filter/select all) -->
         <div class$="{{_getSecondHeaderClass(_enableFilterRow)}}">
           <template is="dom-repeat" items="{{meta}}" as="column" id="filterRepeat">
-            <template is="dom-if" if="{{_isFalse(column.hide)}}">
-              <span class$="{{_getFilterHeaderClass(column)}}">
-                <!-- select all checkbox -->
-                <template is="dom-if" if="{{_isSelectAllColumn(column)}}">
-                  <span class="flex flex--center">
-                    <input id="selectAllCheckbox" type="checkbox"
-                           on-change="_clickSelectAll"/>
-                  </span>
-                </template>
+            <span class$="{{_getFilterHeaderClass(column)}}" hidden$="{{column.hide}}">
```

Comment:
Does this work OK in IE11? I forget but we had weird behaviour previously in IE with the `hidden` attribute. I forget now if it was IE10 or IE11 though...